### PR TITLE
Add React Hooks for customization (part 7)

### PR DESCRIPTION
**Pull request by compulim**
> Related to #2539.

Please merge #2540, #2541, #2542, #2543, #2544 and #2547 before reviewing this one.

## Changelog Entry

-  Resolves [#2539](https://github.com/Microsoft/BotFramework-WebChat/issues/2539), added React hooks for customization, by [@compulim](https://github.com/compulim), in the following PRs:
   -  PR [#2540](https://github.com/microsoft/BotFramework-WebChat/pull/2540): `useActivities`, `useReferenceGrammarID`, `useSendBoxDictationStarted`
   -  PR [#2541](https://github.com/microsoft/BotFramework-WebChat/pull/2541): `useStyleOptions`, `useStyleSet`
   -  PR [#2542](https://github.com/microsoft/BotFramework-WebChat/pull/2542): `useLanguage`, `useLocalize`, `useLocalizeDate`
   -  PR [#2543](https://github.com/microsoft/BotFramework-WebChat/pull/2543): `useAdaptiveCardsHostConfig`, `useAdaptiveCardsPackage`, `useRenderMarkdownAsHTML`
   -  PR [#2544](https://github.com/microsoft/BotFramework-WebChat/pull/2544): `useAvatarForBot`, `useAvatarForUser`
   -  PR [#2547](https://github.com/microsoft/BotFramework-WebChat/pull/2547): `useEmitTypingIndicator`, `usePeformCardAction`, `usePostActivity`, `useSendEvent`, `useSendFiles`, `useSendMessage`, `useSendMessageBack`, `useSendPostBack`
   -  PR [#2548](https://github.com/microsoft/BotFramework-WebChat/pull/2548): `useDisabled`

## Description

Add several post message related hooks:

- `useDisabled`

## Specific Changes

- Added the following hooks:
   - `useDisabled`
- Updated components to use aforementioned hooks

---

-  [x] Testing Added

### CODE REVIEW

File: packages/component/src/SendBox/SuggestedAction.js [hunk 1]
```diff
@@ -31,6 +32,7 @@ const connectSuggestedAction = (...selectors) =>
 
 const SuggestedAction = ({ buttonText, clearSuggestedActions, disabled, displayText, image, text, type, value }) => {
```
Comment (tdurnford) [inline packages/component/src/SendBox/SuggestedAction.js]: Remove disabled from props

File: packages/component/src/SendBox/UploadButton.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import AttachmentIcon from './Assets/AttachmentIcon';
 import connectToWebChat from '../connectToWebChat';
```
Comment (tdurnford) [inline packages/component/src/SendBox/UploadButton.js]: Remove PropTypes Import on line 3

File: packages/component/src/SendBox/SuggestedAction.js (hunk 1 above)
Comment (compulim) [reply packages/component/src/SendBox/SuggestedAction.js]: Fixed.

File: packages/component/src/SendBox/UploadButton.js (hunk 2 above)
Comment (compulim) [reply packages/component/src/SendBox/UploadButton.js]: Fixed.

**Pull request merged**